src/devices/MemoryLCD/Utility.cs

```diff
@@ -0,0 +1,276 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Iot.Device.MemoryLcd
+{
+    internal static class Utility
+    {
+        public static byte GetAgByte(int b)
+        {
+            return s_byteTable[b];
```

Comment:
And please use the proper naming convention for the variable as well `sByteTable` or equivalent.